flink-python/pyflink/table/window.py

```diff
@@ -0,0 +1,482 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+from abc import ABCMeta
+
+from py4j.java_gateway import get_method
+from pyflink.java_gateway import get_gateway
+
+__all__ = [
+    'Tumble',
+    'Session',
+    'Slide',
+    'Over',
+    'GroupWindow',
+    'OverWindow'
+]
+
+
+class GroupWindow(object):
```

Comment:
These three classes can be removed:
SlideWithSizeAndSlideOnTimeWithAlias/TumbleWithSizeOnTimeWithAlias/SlideWithSizeAndSlideOnTimeWithAlias

Reply:
Done.